src/migration/tasks/AddNotificationsFlagsToUsersTask.ts

```diff
@@ -49,24 +49,24 @@ export default class AddNotificationsFlagsToUsersTask extends MigrationTask {
     for (const user of users) {
       // Keep current setting
       user.notifications = {
-        sendSessionStarted: user.notifications.sendSessionStarted ? user.notifications.sendSessionStarted : false,
-        sendOptimalChargeReached: user.notifications.sendOptimalChargeReached ? user.notifications.sendOptimalChargeReached : false,
-        sendEndOfCharge: user.notifications.sendEndOfCharge ? user.notifications.sendEndOfCharge : false,
-        sendEndOfSession: user.notifications.sendEndOfSession ? user.notifications.sendEndOfSession : false,
-        sendUserAccountStatusChanged: user.notifications.sendUserAccountStatusChanged ? user.notifications.sendUserAccountStatusChanged : false,
-        sendSessionNotStarted: user.notifications.sendSessionNotStarted ? user.notifications.sendSessionNotStarted : false,
-        sendUserAccountInactivity: user.notifications.sendUserAccountInactivity ? user.notifications.sendUserAccountInactivity : false,
-        sendPreparingSessionNotStarted: user.notifications.sendPreparingSessionNotStarted ? user.notifications.sendPreparingSessionNotStarted : false,
-        sendNewRegisteredUser: user.notifications.sendNewRegisteredUser ? user.notifications.sendNewRegisteredUser : false,
-        sendUnknownUserBadged: user.notifications.sendUnknownUserBadged ? user.notifications.sendUnknownUserBadged : false,
-        sendChargingStationStatusError: user.notifications.sendChargingStationStatusError ? user.notifications.sendChargingStationStatusError : false,
-        sendChargingStationRegistered: user.notifications.sendChargingStationRegistered ? user.notifications.sendChargingStationRegistered : false,
-        sendOcpiPatchStatusError: user.notifications.sendOcpiPatchStatusError ? user.notifications.sendOcpiPatchStatusError : false,
-        sendSmtpAuthError: user.notifications.sendSmtpAuthError ? user.notifications.sendSmtpAuthError : false,
-        sendOfflineChargingStations: user.notifications.sendOfflineChargingStations ? user.notifications.sendOfflineChargingStations : false,
-        sendBillingSynchronizationFailed: user.notifications.sendBillingSynchronizationFailed ? user.notifications.sendBillingSynchronizationFailed : false,
-        sendCarCatalogSynchronizationFailed: user.notifications.sendCarCatalogSynchronizationFailed ? user.notifications.sendCarCatalogSynchronizationFailed : false,
-        sendEndUserErrorNotification: user.notifications.sendEndUserErrorNotification ? user.notifications.sendEndUserErrorNotification : false,
+        sendSessionStarted: user.notifications?.sendSessionStarted ? user.notifications.sendSessionStarted : false,
```

Comment:
? works only for optional parameter and Angular template

Reply:
It's a typescript syntax extension enabled by default.

> PR again melted with import sorting -> difficult to read, I want only few files to review in this PR

Disable the buggy code extensions that do imports sorting like I've already told you and that will never happen again. Every developer must disable it.

Reply:
> It's a typescript syntax extension enabled by default.
Weird, it didn't work at the time I tested this in ts files but only in template!

Reply:
> Disable the buggy code extensions that do imports sorting like I've already told you and that will never happen again. Every developer must disable it.

I disabled sort-imports already, is it the right one?

I use this one:
![image](https://user-images.githubusercontent.com/25266945/89740688-40c23480-da8b-11ea-9110-3a4e6f1a52c3.png)

Which is related to the Auto Import here:
![image](https://user-images.githubusercontent.com/25266945/89740715-81ba4900-da8b-11ea-931b-5afe7ed5eabe.png)

Reply:
No, it's a code extension named sort-imports, not a code settings related to automatic importing.